### PR TITLE
UINV-78 Invoices - update save and close buttons

### PR DESCRIPTION
**Pull request by AndreiShumski2**
## Purpose
FOLIO is adopting a floating menu bar for the "Save and close" and "Cancel" operations
https://issues.folio.org/browse/UINV-78
## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/invoices/InvoiceLineForm/InvoiceLineForm.js [hunk 1]
```diff
@@ -129,10 +111,26 @@ class InvoiceLineForm extends Component {
...
+        'clickable-update-invoice-line',
+        <FormattedMessage id="ui-invoice.saveAndClose" />,
+        pristine,
+        submitting,
+        handleSubmit,
+        onCancel,
+      )
+      : getPaneFooter(
+        'clickable-create-invoice-line',
+        <FormattedMessage id="ui-invoice.saveAndClose" />,
```
Comment (aliaksei-chumakou) [inline src/invoices/InvoiceLineForm/InvoiceLineForm.js]: if it's actually the same button, maybe it makes sense to leave only one?

File: src/common/utils/getPaneFooter.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import {
+  Button,
+  PaneFooter,
+} from '@folio/stripes/components';
+
+// eslint-disable-next-line import/prefer-default-export
+export const getPaneFooter = (
```
Comment (aliaksei-chumakou) [inline src/common/utils/getPaneFooter.js]: Could it be a functional component? Just to be consistent with our rest React architecture.

Comment (AndreiShumski2) [reply src/common/utils/getPaneFooter.js]: Sure, maybe it's better to create in `stripes-acq-components` for all forms?

Comment (aliaksei-chumakou) [reply src/common/utils/getPaneFooter.js]: Let's start from component in ui-invoice, to see if it leads us somewhere at all

File: src/invoices/InvoiceLineForm/InvoiceLineForm.js [hunk 3]
```diff
@@ -129,10 +111,19 @@ class InvoiceLineForm extends Component {
     const isDisabledToEditAccountNumber = Boolean(isEditPostApproval || (poLineId && accountNumber));
     const isDisabledEditFundDistribution = IS_EDIT_POST_APPROVAL(invoice.id, invoice.status);
 
-    const lastMenu = getLastMenu(handleSubmit, pristine, submitting);
     const paneTitle = initialValues.id
       ? <FormattedMessage id="ui-invoice.invoiceLine.paneTitle.edit" values={{ invoiceLineNumber }} />
       : <FormattedMessage id="ui-invoice.invoiceLine.paneTitle.create" />;
+    const paneFooter = (
```
Comment (NikitaSedyx) [inline src/invoices/InvoiceLineForm/InvoiceLineForm.js]: does it make sense to wrap by memo?

File: test/bigtest/interactors/InvoiceFormInteractor.js [hunk 4]
```diff
@@ -40,7 +40,7 @@ import { OptionListInteractor } from '@folio/stripes-acq-components/test/bigtest
 export default interactor(class InvoiceForm {
   static defaultScope = '#pane-invoice-form';
   isLoaded = isPresent('[class*=paneTitleLabel---]');
-  saveButton = new Interactor('[data-test-button-save-invoice]');
```
Comment (NikitaSedyx) [inline test/bigtest/interactors/InvoiceFormInteractor.js]: imo it's better to keep our selectors

Comment (NikitaSedyx) [reply test/bigtest/interactors/InvoiceFormInteractor.js]: I see, `id` is defined as a prop

imo formfooter component should have interactor

File: src/common/components/FormFooter/FormFooter.js [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+  pristine,
+  submitting,
+  handleSubmit,
+  onCancel,
+}) => {
+  const start = (
+    <FormattedMessage id="ui-invoice.cancel">
+      {(btnLabel) => (
+        <Button
+          id="clickable-close-dialog-footer"
```
Comment (NikitaSedyx) [inline src/common/components/FormFooter/FormFooter.js]: why is it hardcoded if it's kind of common?

File: src/common/components/FormFooter/FormFooter.js [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+          buttonStyle="default mega"
+          onClick={onCancel}
+        >
+          {btnLabel}
```
Comment (NikitaSedyx) [inline src/common/components/FormFooter/FormFooter.js]: `<FormattedMessage id="ui-invoice.cancel" />`?

**Pull request merged**